pipelines/appco/scripts/longhorn-setup.sh

```diff
@@ -575,7 +311,15 @@ main(){
   create_aws_secret
   set -x
 
-  install_backupstores
+  if [[ ${TF_VAR_k8s_distro_name} == "eks" ]]; then
```

Comment:
If this pipeline is specific to test `appco` images and is not required to run on `eks`, this part can be removed.